.github/workflows/deploy-releases.yml

```diff
@@ -176,3 +176,14 @@ jobs:
           npm set //registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}
           cd packages/node && npm publish --access=public
 
+  deploy-consent-tools:
+    uses: ./.github/workflows/deploy-npm.template.yml
```

Comment:
If these packages deploy smoothly, I plan to update all our other jobs to use the template, that way we have a single shared "Deploy to NPM" definition.

It looks like only the `browser` build relies on the `PATH_VERSION` env variable from the previous job logic, so I will make sure to adapt that to the new versioning pattern.